modules/oe_theme_contact_forms/oe_theme_contact_forms.module

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the corporate contact forms.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\contact\Entity\ContactForm;
+
+/**
+ * Implements hook_form_FORM_ID_alter() for contact_message_form().
+ */
+function oe_theme_contact_forms_form_contact_message_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  /** @var \Drupal\Core\Entity\ContentEntityForm $form_object */
+  $form_object = $form_state->getFormObject();
+  /** @var \Drupal\contact\MessageInterface $contact_message */
+  $contact_message = $form_object->getEntity();
+  /** @var \Drupal\contact\Entity\ContactForm $contact_form */
+  $contact_form = ContactForm::load($contact_message->bundle());
+
+  // Display header as ecl paragraph.
+  $form['header']['#markup'] = '<p class="ecl-u-type-paragraph">' . $form['header']['#markup'] . '</p>';
+  // Loged in user name should display in its own line.
+  if ($form['name']['#type'] == 'item') {
+    $form['name']['#field_prefix'] = '<div class="ecl-u-mt-xs">';
+    $form['name']['#field_suffix'] = '</div>';
+  }
+  // Loged in user mail should display in its own line.
+  if ($form['mail']['#type'] == 'item') {
+    $form['mail']['#field_prefix'] = '<div class="ecl-u-mt-xs">';
+    $form['mail']['#field_suffix'] = '</div>';
+  }
+  // Add ecl link classes to the privacy link.
+  $privacy_link = $contact_form->getThirdPartySetting('oe_contact_forms', 'privacy_policy', FALSE);
+  $privacy_title = t('I have read and agree with the <a class="ecl-u-ml-2xs ecl-link ecl-link--default" href="@link" target="_blank">data protection terms</a></label>', [
+    '@link' => $privacy_link,
+  ]);
+  $form['privacy_policy']['#title'] = $privacy_title;
```

Comment:
As anonymous user I can't Send a copy to myself but as logged in user I can. Not sure if it just a confugration.
![image](https://user-images.githubusercontent.com/22004498/85007731-34d98680-b15c-11ea-905b-079da6feafc1.png)

Reply:
thats the way contact forms core is built